### PR TITLE
Refactor breakpoint names

### PR DESCRIPTION
**Pull request by jonnystening**
- Renaming the `sm` (landscape phone and phablet) breakpoint as `xsl` —  which works as either "Extra small landscape" or "Extra small (large)"

- Updated breakpoint name usage and array values notation for components and website

### CODE REVIEW

File: components/alert/examples/00-default.js [hunk 1]
```diff
@@ -34,15 +34,17 @@ function Example({ brand }) {
...
-				This alert needs your attention, but it’s not super important. Lorem ipsum dolor sit amet,
-				consectetur adipisicing elit. Quo dolor provident quasi nisi officia tempore fuga autem,
-				animi iste molestiae, qui omnis doloribus aliquid ipsam rem fugiat veniam voluptatem
-				accusamus! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Est, unde quis,
-				molestias nisi quae voluptates nemo quaerat nihil, consequuntur nobis ratione rerum
-				asperiores eveniet dicta maiores quia nostrum. Pariatur, natus. Lorem ipsum dolor sit amet,
-				consectetur adipisicing elit. Fuga, magnam illum harum consequatur, quo sunt impedit quam
-				minus eaque saepe voluptas corrupti voluptatum, sapiente dolor sequi tempore maxime? Neque,
-				obcaecati. <a href="#">Link</a>
+				<p>
```
Comment (dominikwilkowski) [inline components/alert/examples/00-default.js:37]: You shouldn't have to do this in the alert components

File: helpers/example/components/AppStart.js [hunk 2]
```diff
@@ -42,7 +42,6 @@ export function AppStart({ components, packageName, pkg, version }) {
 							`}
 						/>
 						<Sidebar components={components} brand={brand} setBrand={setBrand} />
-						<p>{version}</p>
```
Comment (dominikwilkowski) [inline helpers/example/components/AppStart.js:45]: Why remove the version?

Comment (jonnystening) [reply helpers/example/components/AppStart.js:45]: It’s in there twice (already in the page heading), that one is breaking the page layout

File: components/alert/examples/00-default.js (hunk 1 above)
Comment (jonnystening) [reply components/alert/examples/00-default.js:37]: May be used with multiple paragraphs 🤷‍♂️

File: helpers/example/components/AppStart.js (hunk 2 above)
Comment (dominikwilkowski) [reply helpers/example/components/AppStart.js:45]: uhm not sure what you mean but the version is not duplicated anywhere I can find???

Comment (jonnystening) [reply helpers/example/components/AppStart.js:45]: Screen Shot 2020-05-19 at 10 37 25 am

Comment (dominikwilkowski) [reply helpers/example/components/AppStart.js:45]: got you

**Pull request merged**